metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/scripts/params/status_params.py

```diff
@@ -52,6 +52,26 @@
 update_table = 'metron_update'
 update_cf = 't'
 
+# Profiler
+metron_profiler_topology = 'profiler'
+profiler_input_topic = config['configurations']['metron-enrichment-env']['enrichment_output_topic']
+profiler_kafka_start = config['configurations']['metron-profiler-env']['profiler_kafka_start']
+profiler_period_duration = config['configurations']['metron-profiler-env']['profiler_period_duration']
+profiler_period_units = config['configurations']['metron-profiler-env']['profiler_period_units']
+profiler_ttl = config['configurations']['metron-profiler-env']['profiler_ttl']
```

Comment:
Do all of these need to be defined here?  These are also defined in params_linux.py and most won't be used for status.

Reply:
Just a side question @merrimanr, how do you know that?  Do you have documentation on what the status scripts are, when and who loads them?  A guide?  Asking for a friend.

Reply:
I found out the hard way, through experience.  The ambari agent runs in a polling loop and calls the "status" function on the command script (profiler_master.py in this case).  Not much documentation out there for this as far as I know.

Reply:
Yeah, I didn't quite entirely realize what the status.py was doing, so I went a bit overboard.  I'll trim them back to just the stuff needed for status checks.